draft-foudil-securitytxt.md

```diff
@@ -514,18 +531,27 @@ the digital signature and checking any available historical records before using
 contained in the file. If the "security.txt" file looks suspicious or compromised,
 it should not be used.
 
+When used for incident response, implementors should be aware that the "security.txt" file
+itself maybe compromised as part of the incident being reported. In such cases, additional methods
+of verifying trust would be needed such as out of band mechanisms including the trust path
+for the PGP signature, DNS-based approaches, etc.
+
+## Redirects {#redirects}
+
 When retrieving the file and any resources referenced in the file, researchers should record
 any redirects since they can lead to a different domain or IP address controlled by an attacker. Further
 inspections of such redirects is recommended before using the information contained within the file.
 
+
+
 ## Incorrect or Stale Information {#stale}
 
 If information and resources referenced in a "security.txt" file are incorrect
 or not kept up to date, this can result in security reports not being received
 by the organization or sent to incorrect contacts, thus exposing possible
 security issues to third parties. Not having a security.txt file may be preferable
-to having stale information in this file. Organizations are also encouraged to
-use the "Expires" field (see {{expires}}) to indicate to researchers when
+to having stale information in this file. Organizations must use
```

Comment:
Shouldn't this be an RFC "MUST"?

```suggestion
to having stale information in this file. Organizations MUST use
```

Reply:
One of the comments received from last call is that the entire security considerations section shouldn't use the capital case terms since it is not normative.

Reply:
Ah yes, good point! I forgot about that. Ignore this suggestion in that case.